pkg/api/api.go

```diff
@@ -8,8 +8,9 @@ type CollectionMetadataSettable interface {
 }
 
 type PaginationData struct {
-	Limit  int `query:"limit" json:"limit" `  //Number of results to return
-	Offset int `query:"offset" json:"offset"` //Offset into the total results
+	Limit   int    `query:"limit" json:"limit" `      //Number of results to return
+	Offset  int    `query:"offset" json:"offset"`     //Offset into the total results
+	OrderBy string `query:"order_by" json:"order_by"` //Order in which to sort results
```

Comment:
We probably need to follow the console dot guidelines:  https://consoledot.pages.redhat.com/docs/dev/developer-references/rest/sorting.html

Reply:
if you wanna hold off on supporting multiple sorts, i think thats fine, but we should probably support the  'sort_by=architecture:asc' syntax

Reply:
Will change, should have looked at the docs first.

Reply:
Don't forget to add a space after // in the comments